CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.10.0
+###### Fixes
+- Remove dependency on regex package as it requires gcc
+- Remove rule [E3507](https://github.com/awslabs/cfn-python-lint/blob/master/docs/rules.md#E3507) because it depends on regex package
+- Support stdin for reading and testing templates
```

Comment:
Don't want to nitpick, but this is more a New feature than a fix

Reply:
Perhaps "officially" mention Python 3.7 support?

Reply:
True :(  Oops. Sorry about that